### PR TITLE
Re-former project

### PR DESCRIPTION
**Pull request by John-Arboleda**
# Re-former: Bare metal forms and helpers

- This project is an exercise for basic form helpers on Ruby on Rails.
- An User has 3 fields: username, email, and password.
- The application allows the creation and edition of users, so only `:new`, `:create`, `:edit`, `:update` methods were use for this app.

### CODE REVIEW

File: app/controllers/users_controller.rb [hunk 1]
```diff
@@ -0,0 +1,34 @@
+class UsersController < ApplicationController
+  def new
+    @user = User.new
+  end
+
+  def create
+    # @user = User.new(username: params[:username], email: params[:email], password: params[:password])
```
Comment (Dipeshtwis) [inline app/controllers/users_controller.rb:7]: Remove this commented code.

File: app/controllers/users_controller.rb [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+    @user = User.find(params[:id])
+
+    if @user.update(user_params)
+      redirect_to(new_user_path)
+    else
+      render(:edit)
+    end
+  end
+
+  def user_params
```
Comment (Dipeshtwis) [inline app/controllers/users_controller.rb:33]: Make `user_params` method private.

**Review by Dipeshtwis — CHANGES_REQUESTED** [review]
# Status:  ♻️ Changes Required ♻️
 Hi @John-Arboleda  and @Maha-Magdy

**You've almost hit all the requirements in one shot. Well done.** Here're the major highlights of this PR:
- Your PR title and description are really descriptive.
- Your README file is looking good.

**However, in order to approve, we have to work a little bit more. Please check my comments below and fix them as appropriate.**

## For Approval

- [x] `user_params` should be a private method.
- [x] Remove commented code.

## [Optional]

- [x] You can create partial for user form and reuse in edit and new view.
- [ ] In the user model you can validate the email format by using either a regular expression or a built-in URI method.

Remember, This is optional and nothing to do with the project approval.

## Please make the required changes and submit another review request when you are done.

(Feel free to contact me on slack by the handle of  **deep** if you have any additional questions).
Happy Coding
[Dipesh Kumar](https://deeptwist.tk) | [Twitter](https://twitter.com/97deepeshkumar)

**Review by GzimAsani — APPROVED** [review]
# STATUS: APPROVED :green_circle: :partying_face: 

## Hello @John-Arboleda and @Maha-Magdy thank you for implementing the required changes. Your project is now completed and you can move to the next project :+1: 

- I think you misunderstood the previous TSE he meant to delete the commented code in your controllers and other places and not the previously used form_tags. But I can understand your confusion

_**PS: Don't forget to submit the project completion form :thought_balloon:**_

# HAPPY CODING :computer: :desktop_computer:

**Pull request merged**